.changeset/eight-turkeys-invent.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': patch
+'@microsoft/atlas-js': patch
+---
+
+Minor update to the form behavior scripts that improves error messaging on radio input
```

Comment:
What do you think about something like "Added form behavior radio group labeling and error handling."?